server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java

```diff
@@ -169,25 +170,23 @@ public TabletServerResourceManager(TabletServer tserver, VolumeManager fs) {
     long maxMemory = acuConf.getAsBytes(Property.TSERV_MAXMEM);
     boolean usingNativeMap = acuConf.getBoolean(Property.TSERV_NATIVEMAP_ENABLED) && NativeMap.isLoaded();
 
-    long blockSize = acuConf.getAsBytes(Property.TSERV_DEFAULT_BLOCKSIZE);
     long dCacheSize = acuConf.getAsBytes(Property.TSERV_DATACACHE_SIZE);
```

Comment:
The sizes are used for sanity checks, but the implementation does not necessarily  have to use these properties.  Also the sanity checks are only for on heap memory.  So think we may want to add something to the BlockCache interface where we can ask it what its max heap usage will be.   

```java
  _iCache = cacheManager.getBlockCache(CacheType.INDEX);
   _dCache = cacheManager.getBlockCache(CacheType.DATA);
   _sCache = cacheManager.getBlockCache(CacheType.SUMMARY);

  //use these values for heap memory over-subscription checks.
  long dCacheSize = _dCache.getMaxHeapSize();
  long iCacheSize = _iCache.getMaxHeapSize();
  long sCacheSize = _sCache.getMaxHeapSize();
```